.travis.yml

```diff
@@ -41,7 +39,6 @@ script:
 - npm run allure:generate
 - npm run functional
 - rollup -c --environment BUILD:production && rollup -c --environment MINIFY,BUILD:production && bundlesize
-- sonar-scanner
```

Comment:
how does this relate to this PR?

Reply:
It allows the build to pass as suggested by @owenleej .

Reply:
I took this out but put it back in through a rebase.